src/commands/config/mod.rs

```diff
@@ -17,10 +17,40 @@ pub fn set_file_mode(file: &PathBuf) {
         .expect("could not set permissions on file");
 }
 
-pub fn global_config(email: String, api_key: String) -> Result<(), failure::Error> {
-    let s = GlobalUser { email, api_key };
+pub fn global_config(api_key: bool) -> Result<(), failure::Error> {
+    let mut user = GlobalUser {
+        email: None,
+        api_key: None,
+        api_token: None,
+    };
 
-    let toml = toml::to_string(&s)?;
+    if !api_key {
+        // Default: use API token.
+        message::info("Looking to use a Global API Key and email instead? Run \"wrangler config --api-key\". (Not Recommended)");
```

Comment:
nit: i don't think the presence of this message makes sense. i think worst case it is confusing, best case it is noise.

Reply:
this was on the PRD--i feel like we can shunt this to after the RC is cut?

Reply:
yah is a nit